pypit/data/settings/settings.keck_deimos

```diff
@@ -387,6 +386,7 @@ settings trace slits tilts method spca
 settings trace slits pca params [3,2,1,0]
 settings trace slits polyorder  3
 settings trace slits sobel mode nearest
+settings trace slits fracignore  0.02   # 0.02 removes star boxes of 40pix size or less (and any real ones too!)
```

Comment:
I think this setting should probably be renamed and also improved so that you can either set the number of pixels (if value > 1), or the fraction (if value < 1).

Reply:
I like that.  Am adding it as an Issue.